examples/neg/namer/issue950/cannot_find_function.effekt

```diff
@@ -0,0 +1,12 @@
+interface Greet { def sayHello(): Unit }
+
+def helloWorld() = try {
+  Hello()
+  // [error] Cannot find a function named `Hello`.
+} with Greet {
+  def sayHello() = { println("Hello!"); resume(()) }
+}
+
+def main() = {
+  helloWorld() // throws:  Cannot find a function named `Hello`.
+}
```

Comment:
Why this test in particular? As far as I can tell, this only tests that a function of a given name does not exist. Wouldn't an easier test like
```scala
def main() = doesNotExist()
```
suffice?